rclcpp/test/rclcpp/executors/test_static_executor_entities_collector.cpp

```diff
@@ -267,7 +267,7 @@ TEST_F(TestStaticExecutorEntitiesCollector, add_remove_node_with_entities) {
   auto client = node->create_client<test_msgs::srv::Empty>("service");
   auto waitable = std::make_shared<TestWaitable>();
 
-  // Adding a subscription with rmw_connext_cpp adds another waitable, so we need to get the
+  // Adding a subscription with could add another waitable, so we need to get the
```

Comment:
there's something missing between `with` and `could`, maybe

```suggestion
  // Adding a subscription could add another waitable, so we need to get the
```